beacon_node/operation_pool/src/lib.rs

```diff
@@ -246,13 +250,13 @@ impl<T: EthSpec> OperationPool<T> {
     pub fn get_slashings(
         &self,
         state: &BeaconState<T>,
-        spec: &ChainSpec,
+        _spec: &ChainSpec,
```

Comment:
This `_spec` is unused. Not a big deal, just pointing it out. Perhaps it's a future boi.

Reply:
Fixed in https://github.com/sigp/lighthouse/pull/2279/commits/6415962290a54c00289cb526aefc5a304e9e94e4